.github/workflows/kuksa_val_docker_arm64.yml

```diff
@@ -0,0 +1,53 @@
+name: kuksa_val_docker arm64
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: [ self-hosted, kuksa-x64 ]
```

Comment:
kuksa-x64 is wrong, needs to be kuksa-arm64  , as we do not have that yet, self-hosted also needs to remain for the time being